PluginJira/API/Utility/EndpointHelperEndpoints/IssuesEndpoint.cs

```diff
@@ -24,23 +24,22 @@ private class IssuesEndpoint : Endpoint
                 // fetch all records
                 var jira = factory.CreateJiraClient(settings);
 
-                string projectKey = "DEMO";
-
+                var projectKey = "DEMO";
```

Comment:
We will want to generalize this so that the user can get issues for any project, or all projects and not just for DEMO